backend/lettercraft/settings.py

```diff
@@ -44,6 +44,7 @@
     "letter",
     "person",
     "source",
+    "space",
```

Comment:
When we make a new deployment, we mustn't forget to give the team members access to this new app!